### PR TITLE
Replace Selenium with BS4

### PR DESCRIPTION
**Pull request by DLzer**
## Why
Initially I ran into a ton of issues with Selenium when spinning this project up. Ended up removing Selenium due to the sheer weight of spinning up a headless browser just to get an `href` from a page alongside the compatability issues it may have.

## How
Removed anything in `mcserver_autoupdater.py` referencing Selenium and replaced it with BS4.

Works in a similar manner though:
- Makes a GET request for the Minecraft downloads page
- Parses the response in BS
- Searches parsed response for matching `aria-label`
```py
for a_tags in soup.findAll('a', attrs={"aria-label":"Download Minecraft Dedicated Server software for Ubuntu (Linux)"}):
```
- Returns matching download link

## TLDR;
- Removed selenium
- Replaces with BS4
- Faster and more predictable responses

### CODE REVIEW

**Comment by ghwns9652** [comment]
Works perfectly. Thank you for your contribution. Honestly, I tried bs4 at first, but couldn't figure out how to make a GET request appropriately. Now I see that it's due to missing header. Selenium worked in much simpler way to make GET request, so I used it instead. However, I think it would be nicer to use bs4 as you wrote.
